src/SystemCommands-MessageCommands/CmdRefactorMenuGroup.class.st

```diff
@@ -0,0 +1,32 @@
+"
+I represent kind of reusable additional method group which users can reuse for grouping set of extra method refactoring commands.
+
+"
+Class {
+	#name : #CmdRefactorMenuGroup,
```

Comment:
In SystemCommands packages the class prefix is expected to be Syc instead of Cmd

Reply:
Ok, I'll rename the class